tools/c7n_azure/tests_azure/azure_common.py

```diff
@@ -410,20 +410,21 @@ def setUp(self):
             self._now_patch.start()
             self.addCleanup(self._now_patch.stop)
 
-        if not self._requires_polling:
-            # Patch Poller with constructor that always disables polling
-            # This breaks blocking on long running operations (resource creation).
-            self._lro_patch = patch.object(msrest.polling.LROPoller, '__init__', BaseTest.lro_init)
-            self._lro_patch.start()
-            self.addCleanup(self._lro_patch.stop)
-
         if self.is_playback():
             if self._requires_polling:
                 # If using polling we need to monkey patch the timeout during playback
                 # or we'll have long sleeps introduced into our test runs
                 Session._old_client = Session.client
                 Session.client = BaseTest.session_client_wrapper
                 self.addCleanup(BaseTest.session_client_cleanup)
+            else:
```

Comment:
Making sure I understand the intent - this ensures poller is not patched during recording?

Reply:
For Long Running Operations we have to manually edit cassettes to remove a bunch of waits